contributor/making-release.md

```diff
@@ -16,12 +52,15 @@ There are number of things that are required before attempting to build a releas
 2. Ensure that you're using the correct major release of Java (check javadoc too).
 3. Ensure that you're building Apache Accumulo with a username that has the same name as your Apache ID (this is due to
    the maven-release-plugin and staging the release candidate).
-4. Update the CHANGES file so that it's in sync with Jira (manual process).
-5. Ensure that you have a texlive distribution installed so you are able to build the documentation.
-6. Have a clean workspace before starting.
+4. Ensure that you have a texlive distribution installed so you are able to build the documentation.
+5. Have a clean workspace before starting.
 
 Given all of this, it's recommended that you only attempt making a release from a GNU/Linux machine.
 
+## Update the CHANGES file 
+
+Update the CHANGES file so that it's in sync with Jira (manual process).
+
 ## Triage issues.
 
 Before creating a release candidate, all open issues with a fix version of the
```

Comment:
I don't think we need wording about JIRA or GitHub transition. 2.0 is the last release with issues tagged in JIRA still... we can just drop this wording, since it adds unnecessary verbosity.